### PR TITLE
Updated signup card frontend styles for contained split layout

### PR DESCRIPTION
**Pull request by sanne-san**
Refs https://github.com/TryGhost/Team/issues/3459

### CODE REVIEW

File: packages/kg-default-nodes/lib/nodes/signup/SignupParser.js [hunk 1]
```diff
@@ -50,21 +50,21 @@ export class SignupParser {
                             const disclaimer = domNode.querySelector('p')?.textContent || '';
                             const backgroundImageSrc = layout === 'split'
                                 ? domNode.querySelector('.kg-signup-card-image')?.getAttribute('src')
-                                : domNode.querySelector('.kg-signup-card-container')?.getAttribute('style')?.match(/url\((.+)\)/)?.[1] || '';
-                            const backgroundColor = domNode.querySelector('.kg-signup-card-container')?.style.backgroundColor || '';
+                                : domNode.querySelector('.kg-signup-card-text')?.getAttribute('style')?.match(/url\((.+)\)/)?.[1] || '';
```
Comment (SimonBackx) [inline packages/kg-default-nodes/lib/nodes/signup/SignupParser.js:53]:
```suggestion
                                : domNode.querySelector('.kg-signup-card')?.getAttribute('style')?.match(/url\((.+)\)/)?.[1] || '';
```

File: packages/kg-default-nodes/lib/nodes/signup/SignupParser.js [hunk 2]
```diff
@@ -50,21 +50,21 @@ export class SignupParser {
...
+                            const backgroundColor = domNode.querySelector('.kg-signup-card-text')?.style.backgroundColor || '';
```
Comment (SimonBackx) [inline packages/kg-default-nodes/lib/nodes/signup/SignupParser.js:54]:
```suggestion
                            const backgroundColor = domNode.querySelector('.kg-signup-card')?.style.backgroundColor || '';
```

File: packages/kg-default-nodes/lib/nodes/signup/SignupParser.js [hunk 3]
```diff
@@ -50,21 +50,21 @@ export class SignupParser {
...
                             const buttonColor = domNode.querySelector('.kg-signup-card-button')?.style.backgroundColor || '';
                             const buttonText = domNode.querySelector('.kg-signup-card-button-default')?.textContent?.trim() || 'Subscribe';
                             const buttonTextColor = domNode.querySelector('.kg-signup-card-button')?.style.color || '';
                             const textColor = domNode.querySelector('.kg-signup-card-success')?.style.color || '';
-                            const alignment = domNode.querySelector('.kg-signup-card-container')?.classList.contains('align-center') ? 'center' : 'left';
+                            const alignment = domNode.querySelector('.kg-signup-card-text')?.classList.contains('align-center') ? 'center' : 'left';
```
Comment (SimonBackx) [inline packages/kg-default-nodes/lib/nodes/signup/SignupParser.js:59]: Note to self: lets rename this to kg-align-center or something similar

File: packages/kg-default-nodes/lib/nodes/signup/SignupParser.js [hunk 4]
```diff
@@ -50,21 +50,21 @@ export class SignupParser {
...
                             const successMessage = domNode.querySelector('.kg-signup-card-success')?.textContent?.trim() || '';
                             const labels = [...domNode.querySelectorAll('input[data-members-label]')].map(input => input.value);
 
-                            const isAccentBackground = domNode.querySelector('.kg-signup-card-container')?.classList?.contains('kg-style-accent') ?? false;
+                            const isAccentBackground = domNode.querySelector('.kg-signup-card-text')?.classList?.contains('kg-style-accent') ?? false;
```
Comment (SimonBackx) [inline packages/kg-default-nodes/lib/nodes/signup/SignupParser.js:63]:
```suggestion
                            const isAccentBackground = domNode.querySelector('.kg-signup-card')?.classList?.contains('kg-style-accent') ?? false;
```

File: packages/kg-default-nodes/lib/nodes/signup/SignupParser.js (hunk 3 above)
Comment (sanne-san) [reply packages/kg-default-nodes/lib/nodes/signup/SignupParser.js]: ✅

**Pull request merged**